code/modules/organs/organ_internal.dm

```diff
@@ -122,9 +118,25 @@
 	var/datum/limb/parent = owner.get_limb(parent_limb)
 	if (!silent)
 		owner.custom_pain("Something inside your [parent.display_name] hurts a lot.", 1)
+	set_organ_status()
+
+/// Set the correct organ state
+/datum/internal_organ/proc/set_organ_status()
+	if(damage > min_broken_damage)
+		if(organ_status != ORGAN_BROKEN)
+			organ_status = ORGAN_BROKEN
+		return TRUE
```

Comment:
are the TRUE returns meaningful here? seems like it happens in all cases except where the organ was healthy both previously and now